.github/workflows/publish_alpha.yml

```
@@ -0,0 +1,137 @@
+          LABELS=$(echo '${{ toJson(github.event.pull_request.labels) }}' | jq -r '.[].name')
+
+          # Handle the case where there are no labels
+          if [ -z "$LABELS" ]; then
+            echo "No labels found on the pull request."
+          else
+            echo "Labels: $LABELS"
+          fi
+
+      - name: Determine version bump
+        id: version_bump
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)